bundles/binding/org.openhab.binding.mystromecopower/src/main/java/org/openhab/binding/mystromecopower/internal/MyStromEcoPowerBinding.java

```diff
@@ -0,0 +1,352 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mystromecopower.internal;
+
+import static org.quartz.TriggerBuilder.newTrigger;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.Date;
+import java.util.Dictionary;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.openhab.binding.mystromecopower.MyStromEcoPowerBindingProvider;
+
+import org.apache.commons.lang.StringUtils;
+import org.openhab.core.binding.AbstractActiveBinding;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.openhab.binding.mystromecopower.internal.api.*;
+import org.openhab.binding.mystromecopower.internal.api.mock.MockMystromClient;
+import org.openhab.binding.mystromecopower.internal.api.model.MystromDevice;
+import org.openhab.binding.mystromecopower.internal.util.ChangeStateJob;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedService;
+import org.quartz.JobBuilder;
```

Comment:
Please use OpenHab-internal Thread Pools instead of configuring Quartz jobs yourself...

From the HowToContribute:

Creation of threads must be avoided. Instead, resort into using existing schedulers which use pre-configured thread pools. If there is no suitable scheduler available, start a discussion in the forum about it rather than creating a thread by yourself.